config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
+  match ':status', to: 'errors#show', via: :all, constraints: { status: /\d{3}/ }
```

Comment:
Line is too long. [81/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.